src/rpc/rawtransaction.cpp

```diff
@@ -310,13 +310,25 @@ UniValue getrawtransaction(const UniValue& params, bool fHelp)
             + HelpExampleCli("getrawtransaction", "\"mytxid\"")
             + HelpExampleCli("getrawtransaction", "\"mytxid\" 1")
             + HelpExampleRpc("getrawtransaction", "\"mytxid\", 1")
+            + HelpExampleRpc("getrawtransaction", "\"mytxid\" true")
+            + HelpExampleRpc("getrawtransaction", "\"mytxid\", true")
         );
 
     uint256 hash = ParseHashV(params[0], "parameter 1");
 
     bool fVerbose = false;
-    if (params.size() > 1)
-        fVerbose = (params[1].get_int() != 0);
+    if (params.size() > 1 && !params[1].isNull()) {
+        if (params[1].isNum() && params[1].get_int() == 1) {
+            fVerbose = true;
+        }
+        else if(params[1].isBool() && params[1].isTrue()) {
+            fVerbose = true;
+        }
+        else {
```

Comment:
what would happen here if params[1] is 0 or false?

Reply:
Yep. Only raise the error if the type is not int or bool and not dependent on value